packages/frontend/src/pages/scaling-activity/props/getProps.ts

```diff
@@ -1,14 +1,27 @@
+import { ApiActivity } from '@l2beat/types'
+
 import { Config } from '../../../build/config'
 import { getFooterProps, getNavbarProps } from '../../../components'
+import { getPercentageChange } from '../../../utils/utils'
 import { Wrapped } from '../../Page'
 import { ActivityPageProps } from '../view/ActivityPage'
 import { getActivityView } from './getActivityView'
 import { getPageMetadata } from './getPageMetadata'
 
-export function getProps(config: Config): Wrapped<ActivityPageProps> {
+export function getProps(
+  config: Config,
+  apiActivity: ApiActivity,
+): Wrapped<ActivityPageProps> {
+  const txCount = apiActivity.combined.data.at(-1)?.[1] ?? 0
+  const txCountSevenDaysAgo = apiActivity.combined.data.at(-7)?.[1] ?? 0
+  const sevenDayChange = getPercentageChange(txCount, txCountSevenDaysAgo)
+
   return {
     props: {
       navbar: getNavbarProps(config),
+      txCount: txCount.toString(),
+      sevenDayChangeTxCount: sevenDayChange,
+      apiEndpoint: '/api/activity.json',
```

Comment:
In the future I would imagine that we'd have `/api/tvl/{project}.json` and `/api/activity/{project}.json`

Reply:
should I change anything here then?

Reply:
I think it should be in another PR